templates/llm-router/README.ipynb

```diff
@@ -0,0 +1,1038 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Background\n",
```

Comment:
A few open question:

What are some concerns about runtime of injecting a router at the for-front of this decision? How does the latency of this model impact the architecture choice of it?